test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -253,9 +253,9 @@ Date: REGEX(.*)
     "measuring_interval_in_secs": "REGEX(\d+)",
     "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)",
     "statisticsRequests": "1",
-    "subCache": "REGEX([0-9a-f, ]{50})",
-    "subCacheInserts": "2",
-    "subCacheItems": "2",
+    "subCache": "REGEX([0-9a-f, ]{76})",
```

Comment:
This thing here is a little bit of debugging, I would go further and REMOVE if from here and implement it in GET /subCache or something similar.

But, this is for the future